ext/ldap/ldap.c

```diff
@@ -2235,7 +2235,7 @@ static void php_ldap_do_modify(INTERNAL_FUNCTION_PARAMETERS, int oper, int ext)
 	unsigned int attribute_index = 0;
 	ZEND_HASH_FOREACH_STR_KEY_VAL(attributes_ht, attribute, attribute_values) {
 		if (attribute == NULL) {
-			php_error_docref(NULL, E_WARNING, "Unknown attribute in the data");
+			zend_argument_value_error(3, "must be an associative array of attribute => values");
```

Comment:
The plural use is a bit strange here. I would write `attribute => value` or `attributes => values` but not mix them. WDYT?

Reply:
The thing is for a single attribute there might be multiple values attached to it